portfolio.html

```diff
@@ -10,11 +10,11 @@
 </head>
 <body>
 
-	<nav class="navigationBar">
-			<a href="#topJump"><button>Home</button></a>
-			<a href="#aboutMeJump"><button>About</button></a>
-			<a href="#portfolioJump"><button>Portfolio</button></a>
-			<a href="#footerJump"><button>Socials</button></a>
+	<nav class="navigationBar fontStyleHead">
+			<a href="#topJump"><button class="fontStyleHead">Home</button></a>
+			<a href="#aboutMeJump"><button class="fontStyleHead">About</button></a>
+			<a href="#portfolioJump"><button class="fontStyleHead">Portfolio</button></a>
+			<a href="#footerJump"><button class="fontStyleHead">Socials</button></a>
```

Comment:
Why are you using buttons within a link tag?

Reply:
I am using them as I wanted buttons instead of links, but to get them to work I need to wrap them in link tags so they go to the required location. This is also the way I found online.